Module1/todo_app/app.py

```diff
@@ -38,38 +40,31 @@ def index():
 
         return render_template('trello.html', view_model = view_model, BOARD_LIST = BOARD_LIST)  
 
-    @app.route('/trello_list', methods=['GET'])
-    def cards_list():
-    
-        # BOARD_LIST = trello_instance.lists_on_board()
-        card_items = trello_instance.get_cards()
-        return render_template('trello.html', card_items = card_items, BOARD_LIST = BOARD_LIST)  
-
     @app.route('/trello_list', methods=['POST', 'GET'])
     def new_card_submit():
+
         if request.method == "POST": 
             card_name = request.form.get("card_title")
             card_desc = request.form.get("card_description")
             card_due_date = request.form.get("card_due_date")
 
         if card_name: 
             trello_instance.new_card(card_name, BOARD_LIST['To Do'], card_due_date, card_desc)
-            card_items = trello_instance.get_cards()
-            return render_template('trello.html', card_items = card_items, BOARD_LIST = BOARD_LIST)
+            return redirect('/')
         else:
-            card_items = trello_instance.get_cards()
-            return render_template('trello.html', card_items = card_items, BOARD_LIST = BOARD_LIST)
+            return redirect('/')
```

Comment:
Really minor, but you'll sometimes see people prefer the pattern below when the final action is shared, to make it clear that all routes lead to the same result:
```python
        if card_name: 
            trello_instance.new_card(card_name, BOARD_LIST['To Do'], card_due_date, card_desc)
        return redirect('/')
```